src/constants/markets.ts

```diff
@@ -29,6 +29,10 @@ export type MarketData = {
   trades24H?: Nullable<number>;
   volume24H?: Nullable<number>;
   tags?: Nullable<string[]>;
+
+  // whether market is favorited;
+  // `isPinned` is also used for Table component to keep it pinned
+  isPinned: boolean;
```

Comment:
instead of `isPinned` why not match with the feature as `isFavorited`?

Reply:
i was debating that -- think it's just the extra layer of mapping isFavorite -> isPinned that feels unnecessary, but i can add that back